pyth-sdk/README.md

```diff
@@ -81,3 +81,49 @@ println!("0.1 BTC and 0.05 ETH are worth: ({} +- {}) x 10^{} USD",
 ```
 
 This operation can be useful for pricing, e.g., an LP token that is backed by two underlying currencies.
+
+## Liquidity-related Pricing Measures
+
+A variety of measures should be taken for protocols looking to mitigate liquidity-related risks. To limit deposits and borrows, protocols could simply derive their limit from the off-chain liquidity estimates and then store those values in contract state. No explicit interaction with the Pyth SDK would be needed in this simple case.
+
+To adjust the price at which collateral is valued based on the liquidity information, a protocol can combine the current Pyth price and their estimate of liquidity:
+
+```rust
+let btc_usd: Price = ...;
+let deposits: u64 = ...;
+let deposits_endpoint: u64 = ...;
+let rate_discount_initial: u64 = ...;
```

Comment:
suggest adding like a code comment above this line to the effect of:

// rate_discount_initial and _final are percentages, expressed in fixed point as `x * 10^discount_exponent`.
// E.g. 50 with discount_exponent = -2 is 50%.